GOVERNANCE.md

```diff
@@ -88,3 +88,12 @@ As seen above, the different teams under the Node.js Web Team umbrella are respo
 Following this line of thought, the Web Infra Team is responsible for the technical aspects of the Node.js Website (Infrastructure, Framework, CI/CD, etc); The Website Team is responsible for the day-to-day development of the Node.js Website; The UX and Design Team advise on Design Matters and the Web Standards Team advise on best-practices for Web APIs and Web Technologies/Standards.
 
 But above all, the Web Team should work together to better the Web Experience for Node.js, aiming to provide the best experience for Node.js users.
+
+## Blog post publishing guidelines
+
+The following guidelines apply to all blog posts except minor or patch release announcements:
+
+1. each blog post _must_ have a target publishing date and time. If scheduled publishing is not possible, _the author_ (if privileged enough) or another delegated member would be responsible for landing.
+2. each blog post _must_ be approved by @nodejs/tsc members _and_ OpenJS Foundation marketing staff.
```

Comment:
If we can pull together a nodejs/marketing team that has members from both the TSC and the foundation marketing staff, would it be enough here to say that the nodejs/marketing team must approve it... then that team can work out whatever process makes the most sense to it.

Reply:
does https://github.com/orgs/nodejs/teams/social suffice?

Reply:
@jasnell if both the TSC and OpenJS staff are members of the team, technically the TSC could approve this without asking them. This has a different meaning.

Reply:
Fair. If the new @nodejs/marketing team, then, is made up of Foundation marketing staff, I agree that requiring an approval from at least one person in both the @nodejs/tsc and @nodejs/marketing teams is ideal.